synapse/http/matrixfederationclient.py

```diff
@@ -98,10 +99,6 @@
 
 _next_id = 1
 
-
-QueryArgs = Dict[str, Union[str, List[str]]]
```

Comment:
We pretty much just had two implementations of the same thing?

Reply:
Correct: one from #8806 and #8372. I think yourself and Rich basically ended up at the same solution.